AElf.Types.CSharp.Tests/ConversionExtensionTests.cs

```diff
@@ -10,20 +10,38 @@ namespace AElf.Types.CSharp.Tests
 {
     public class ConversionExtensionTests
     {
-        [Fact(Skip = "Not passed due to some reason.")]
-        public void Deserialize_To_Bool()
+        [Fact]
+        public void Deserialize_From_ByteString_To_Bool()
         {
-            var message = true.ToPbMessage();
-            var bs = ByteString.FromStream(new MemoryStream(message.ToByteArray()));
-            var boolValue = (bool)bs.DeserializeToType(typeof(bool));
-            boolValue.ShouldBeTrue();
-            
-            var message1 = false.ToPbMessage();
-            var bs1 = ByteString.FromStream(new MemoryStream(message1.ToByteArray()));
-            var boolValue1 = (bool)bs1.DeserializeToType(typeof(bool));
-            boolValue1.ShouldBeFalse();
+            var boolValue = new BoolValue(){ Value = true };
+            var bs = boolValue.ToByteString();
+            var returnObj = (bool)bs.DeserializeToType(typeof(bool));
+            returnObj.ShouldBeTrue();
+
+            boolValue.Value = false;
+            bs = boolValue.ToByteString();
+            returnObj = (bool)bs.DeserializeToType(typeof(bool));
+            returnObj.ShouldBeFalse();
+        }
+
+        [Fact(Skip = "Deserialize data is not correct.")]
+        public void Deserialize_From_ByteString_To_Int()
+        {
+            var intValue = new IntValue() { Value = 36 };
+            var bs = intValue.ToByteString();
+            var returnObj = (int)bs.DeserializeToType(typeof(int));
```

Comment:
IntValue cannot deserialize to int，it should be IntValue

Reply:
updated.